src/Views/Pointing.vala

```diff
@@ -84,19 +69,41 @@ public class MouseTouchpad.PointingView : Granite.SimpleSettingsPage {
         content_area.attach (new SettingLabel (_("Pointer size:")), 0, 0);
         content_area.attach (cursor_size_grid, 1, 0, 3);
 
-        content_area.attach (reveal_pointer_label, 0, 1);
-        content_area.attach (reveal_pointer_switch, 1, 1, 3);
-        content_area.attach (locate_pointer_help, 1, 2, 3);
+        var reveal_pointer_schema = SettingsSchemaSource.get_default ().lookup (
```

Comment:
I'm not sure that's enough. The schema `org.gnome.settings-daemon.peripherals.mouse` still exists, but only has the `double-click` and `drag-threshold` keys (checked with gnome-settings-daemon 3.34.2).

Reply:
So I guess only the `locate-pointer` key is gone.